### PR TITLE
Milestone 4: School Library | Add Basic UI

### PR DESCRIPTION
**Pull request by michael-duke**
## School Library milestone 4 : Add basic UI

In this milestone I have:

- Built interactive console apps.
- Ran a program using the command line.
- Create an `app.rb` file that will serve as your console app entry point. Implement methods that do the following:
  - List all `books`.
  - List all `people`.
  - Create a `person` (`teacher` or `student`,).
  - Create a `book`.
  - Create a `rental`.
  - List all `rentals` for a given `person's id`.
- Defined the entry point in my `main.rb` file, implemented a method called `main` that is invoked at the end of my file which; 
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - Let the user quit the app

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+  def create_person
+    puts 'Which do you want to create? A student (press 1) or a teacher (press 2)?'
+    choice = gets.chomp.to_i
+    case choice
+    when 1
+      register_student
+    when 2
+      register_teacher
+    end
```
Comment (GzimAsani) [inline app.rb]: - [x] If a user enters something else beside 1 or 2 let them know that they input a incorrect number

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions
-N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- 

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: app.rb (hunk 1 above)
Comment (michael-duke) [reply app.rb]: Thanks @GzimAsani , I will make that fix

**Comment by michael-duke** [comment]
Thank you @GzimAsani for the feedback, happy coding 😄 🚀

**Pull request merged**